content/tutorials/getting-started-with-solid/components/BasicBookshelf.tsx

```diff
@@ -1,9 +1,9 @@
 import { createSignal, For, JSX, Setter } from "solid-js";
 
-type Book = {
+interface Book {
```

Comment:
This was auto-fixed by [`@typescript-eslint/consistent-type-definitions`](https://typescript-eslint.io/rules/consistent-type-definitions), which is set to prefer interfaces in the strict config. If you prefer types you can change it to prefer types and run `npm run lint -- --fix` again.